takwimu/templates/takwimu/_includes/sections/explore/countries.html

```diff
@@ -90,7 +90,7 @@
         <div class="col" style="border-right: none;">
             <a href="#" data-toggle="tooltip" data-placement="top" title="Coming Soon">
                 <img class="svg" src="{% static 'img/flags/zambia.svg' %}" alt="Zambia">
-                <span class="text-uppercase">Zambia</span>
+                <small class="text-uppercase mt-3 d-flex justify-content-center">Zambia</small>
```

Comment:
Why the addition of flex utils?

Reply:
@DavidLemayian to center the individual items, specifically vertically